File: PHP/mysql/pro12.php
<?php
    /*
        PHP MySQL Select Data
        Select Data From a MySQL Database
        The SELECT statement is used to select data from one or more tables:
    */ 

    $hostname = "localhost";
    $username = "root";
    $password = "";
    $database = "mydb";

    try{
        $connection = new mysqli($hostname, $username, $password, $database);

        if($connection->connect_error){
            throw new Exception("Error while connecting Database Server ".$connection->connect_error);
        }else{
            $sqlquery = "select * from students";

            $result = $connection->query($sqlquery);

            //print_r($result);

            // check how many rows receive in result
            if($result->num_rows > 0){
                // run loop and get one by one rows as associative array
                while($row = $result->fetch_assoc()){
                    echo "<hr> Welcome {$row['fname']} {$row['lname']} to {$row['city']} of {$row['state']} your roll no. is {$row['roll']} and contact details are {$row['email']} {$row['phone']}";
                    //print_r($row);
                }
            }else{
                echo "<hr> No Data Found in Table";
            }
        }
    }catch(Exception $err){
        echo "<hr> Error is $err";
    }finally{
        $connection->close();
    }
?>

File: Python/mySql/pro12.py
from sys import prefix

import mysql.connector

try:
    connection = mysql.connector.connect(host="localhost", user="root",
                                         passwd="", database="pythondb")
    cursor = connection.cursor()
    result = cursor.execute("select * from students")
    data = cursor.fetchone()
    print(data)
    data = cursor.fetchone()
    print(data)
    data = cursor.fetchone()
    print(data)
except Exception as err:
    print("Error is ",err)
finally:
    connection.close()